sdks/java/core/src/main/java/org/apache/beam/sdk/util/PubsubApiaryClient.java

```diff
@@ -73,6 +73,11 @@ public PubsubClient newClient(
           .build();
       return new PubsubApiaryClient(timestampLabel, idLabel, pubsub);
     }
+
+    @Override
+    public String toString() {
+      return "Apiary";
```

Comment:
decided it was nicer to include those in the display data in PubsubUnboundedSink.WriterFn.populateDisplayData